src/richie/apps/courses/models/course.py

```diff
@@ -683,6 +701,12 @@ class CourseRun(TranslatableModel):
         choices=lazy(lambda: ALL_LANGUAGES, tuple)(),
         help_text=_("The list of languages in which the course content is available."),
     )
+    enrollment_count = models.PositiveIntegerField(
+        _("enrollment count"),
+        blank=True,
+        null=True,
```

Comment:
Maybe it should be required and default to 0 so we never display a null value?

Reply:
Ok.
For sure we aren't going to show zero enrollments to the user, so for the users the null or zero is the same, it wan't display any enrollment information.